CHANGELOG.md

```diff
@@ -3,6 +3,84 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.14.0
+
+_Sep 14, 2023_
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Fix `YearCalendar` and `MonthCalendar` a11y (#10312) @LukasTy
```

Comment:
```suggestion
- 🎁 Fix `YearCalendar` and `MonthCalendar` accessibility (#10312) @LukasTy
```

Spell it out the first time ([as per Writing style guide](https://www.notion.so/mui-org/Writing-style-guide-2a957a4168a54d47b14bae026d06a24b?pvs=4#d55f3492e3df426d9d5f64fee4a17d9f))?

Reply:
I'd argue if the mentioned rule applies here (IMHO, I'd put the a11y right beside l10n, which we use as a prefix for changes/PRs 🙈 ), but it won't hurt. 👌